### PR TITLE
Use the new overwrite stacks feature 

### PR DESCRIPTION
**Pull request by chregu**

### CODE REVIEW

File: src/Image.php [hunk 1]
```diff
@@ -284,20 +284,30 @@ public function listOperations()
      * @param array  $stackOperations Stack operations
      * @param string $organization    Optional organization name
      * @param array  $stackOptions    Stack options
+     * @param bool   $overwrite       If an existing stack should be overwritten
      *
      * @return Stack
      */
-    public function createStack($stackName, array $stackOperations, $organization = '', array $stackOptions = [])
+    public function createStack(
+                                $stackName,
```
Comment (Tobion) [inline src/Image.php]: that indention looks awkward. normally it's just 4 spaces and not based on the function name length.

File: src/Image.php [hunk 2]
```diff
@@ -284,20 +284,30 @@ public function listOperations()
...
-
+        $queryString = '';
+        if ($overwrite) {
+            $queryString = '?overwrite=true';
+        }
         $contents = $this
             ->call(
                 'PUT',
-                implode('/', [self::STACK_RESOURCE, $this->getOrganization($organization), $stackName]),
+                implode('/', [self::STACK_RESOURCE, $this->getOrganization($organization), $stackName]) . $queryString,
```
Comment (Tobion) [inline src/Image.php]: I'd recommend to use the `query` request option instead as it handles encoding etc, i.e. `['json' => $stackData, 'query' => ['overwrite' => 'true']]`

File: src/Image.php (hunk 1 above)
Comment (chregu) [reply src/Image.php]: Fixed

File: src/Image.php (hunk 2 above)
Comment (chregu) [reply src/Image.php]: Fixed. Thanks for the hint

**Pull request merged**